### ISSUE TITLE
Facebook login stuck at 'signing in' on Android

### ISSUE THREAD

**Issue by ritazagoni** [guardianproject/ChatSecureAndroid#738]
I use Andriod 4.1.2. I try to login to Facebook chat with username@chat.facebook.com using the Jabber/XMPP login. The login process gets stuck at 'Signing in'.

**Comment by n8fr8** [comment]
Unfortunately, Facebook has recently disabled access to their servers through the open XMPP protocol. We are hoping to move to their new API, but we are not yet sure if it is possible. :(

**Comment by ritazagoni** [comment]
That's exactly what I was afraid of. Was there any official communication
about this from Facebook? Or unofficial...? I read about their response
about Crypter stating that they didn't change anything
http://www.digitaltrends.com/social-media/facebook-puts-an-end-to-message-encryption-app-crypter/.
And all at once Chatsecure and Jitsi Facebook connection don't work either.
Thanks!

**Comment by n8fr8** [comment]
Yes they did here quite awhile ago: https://developers.facebook.com/docs/chat

We just didn't believe they would actually follow through, since their April 30th, 2015 deadline passed and it was still working.

**Comment by keshavkaul** [comment]
@n8fr8 Hi, i was looking around and found this nodejs repo [facebook-chat-api](https://github.com/Schmavery/facebook-chat-api). This repo makes it possible to connect to facebook without using the deprecated chat api. Maybe this will be of some help.
